src/AnyStatus.Plugins/Widgets/Metrics/ThreadCount/ThreadCount.cs

```diff
@@ -0,0 +1,25 @@
+﻿using AnyStatus.API;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace AnyStatus
+{
+    [DisplayColumn("Metrics")]
+    [DisplayName("Thread Count")]
+    [Description("Shows the number of running CPU threads")]
+    public class ThreadCount : Sparkline, ISchedulable
+    {
+        public ThreadCount()
+        {
+            Symbol = "";
+            Interval = 10;
+            Units = IntervalUnits.Seconds;
+            Name = "Thread Count";
+        }
```

Comment:
Does not specifying a `MaxValue = 100;` here make the Sparkline only ever be a straight line?
If so does that mean we can't have a sparkline for this plugin?
The same goes for AnyStatus/Plugins#15

Reply:
I don't remember exactly but I think it's not a requirement.
It is needed, in some cases like CPU Usage, to scale the chart, otherwise the "max value" is the highest known value.

Reply:
Side note, I am thinking of changing the Sparkline class to ISparkline interface and use it similarly to other features such as ISchedulable, IStartable, etc.